$NetBSD: patch-ad,v 1.1 2008/01/29 04:18:45 bjs Exp $

Register objects used by libXft. (patch from freedesktop.org GIT)
 
libXft was using XFT_MAX_GLYPH_MEMORY without first registering
with fontconfig.

--- src/xftname.c.orig	2006-12-07 05:17:45.000000000 -0500
+++ src/xftname.c
@@ -28,6 +28,7 @@ static const FcObjectType	_XftObjectType
     { XFT_CORE,		FcTypeBool, },
     { XFT_XLFD,		FcTypeString, },
     { XFT_RENDER,	FcTypeBool, },
+    { XFT_MAX_GLYPH_MEMORY, FcTypeInteger, },
 };
 
 #define NUM_OBJECT_TYPES    (sizeof _XftObjectTypes / sizeof _XftObjectTypes[0])
